sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java

```diff
@@ -6,6 +6,9 @@
 import com.azure.storage.common.sas.SasProtocol;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
```

Comment:
Unused imports? Not seeing any code changes other than this in the class but that could be from return type changes?

Reply:
Fixed